<?xml version="1.0" ?>
<sdf version="1.7">
  <model name="camera">
    <static>1</static>
    <link name="link">
      <pose>50.0 50.0 30.0 0 0.45 -2.35</pose>
      <inertial>
        <mass>0.1</mass>
      </inertial>
      <collision name="collision">
        <geometry>
          <box>
            <size>0.1 0.1 0.1</size>
          </box>
        </geometry>
      </collision>
      <visual name="visual">
        <geometry>
          <box>
            <size>0.5 0.5 0.5</size>
          </box>
        </geometry>
        <material>
          <script>Gazebo/Red</script>
        </material>
      </visual>
      <sensor name="arvc_cam" type="camera">
        <camera>
          <horizontal_fov>1.047</horizontal_fov>
          <image>
            <width>1920</width>
            <height>1080</height>
          </image>
          <clip>
            <near>0.1</near>
            <far>1000</far>
          </clip>
          <save enabled="false">
            <path>/media/arvc/data</path>
          </save>
        </camera>
        <always_on>1</always_on>
        <update_rate>100</update_rate>
        <visualize>false</visualize>
      </sensor>
    </link>
  </model>
</sdf>